### ISSUE TITLE
Incompatibility With The Dawn Era

### ISSUE THREAD

**Issue by Billnotic** [Provismet/ProviHealth#15]
### Type of Error

Incorrect Behaviour

### Mod List

The Dawn Era
Provi's Health Bars

### Description

The Dawn Era's mobs use a separate rendering method labelled in their code as renderFinal(), this means that this mod's healthbar rendering is never injected and those mobs don't have any.

I'd love to see a compatibility patch or something down the line.

### Mod Version

1.2.2+1.20.1

### Minecraft Version

1.20.1

### Mod Loader

Other (specify in desc)

### Steps to Reproduce

```bash
Running in Forge with Sinytra Connector, but this should still occur in Fabric.

Download both mods, start a world and look at a dinosaur (hostile or not or injured or whatever, doesn't matter).
```

**Comment by Provismet** [comment]
This is not feasible for a number of reasons:
- The Dawn Era is an All Rights Reservered and closed source mod.
  - I cannot acquire the source code and see the problem myself. Mods should always be open-source.
  - There is also no public package available, meaning I can't even reference the code without relying on Modrinth's Maven (which is bad, and they even discourage its use).
  - Directly connecting to an All Rights Reserved mod without permission is legally dubious at best.
  - Frankly I'm impressed you even know what their functions are called.
- It's a Forge mod with no Fabric port.
  - I can't add it into my dev environment.
- My mod actually applies its hook at the base of the renderer hierarchy, in the EntityRenderer itself. This means one of two things:
  - The dinosaurs are using a completely different renderer hierarchy with no callbacks to the entity renderer (extremely questionable decision).
  - Or that they are not classified as *living entities* (also extremely questionable).

It's also possible that oddities are occuring due to running a Fabric mod on Forge, but given you only mentioned dinosaurs, I'll assume other mobs are fine and Sinytra Connector is working properly.

### tl;dr
No, I can't and won't add support for this.

And frankly, if someone decides to close-source their mod then I believe the onus is on them to fix every compatibility issue that arises from such a choice.

That being said, if someone were to somehow make a PR that fixes the compatibility without introducing new dependencies into Provi's Health Bars, then I would merge it.
